src/main/java/seedu/address/logic/Messages.java

```diff
@@ -19,6 +19,8 @@ public class Messages {
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
     public static final String MESSAGE_DUPLICATE_FIELDS =
                 "Multiple values specified for the following single-valued field(s): ";
+    public static final String MESSAGE_INVALID_NUSNET_FOR_MARK =
+            "There is no student of this nusNet!";
```

Comment:
I think the message name can be more general, so that we can reuse it for invalid nus net ids in other commands?
```suggestion
    public static final String MESSAGE_INVALID_NUSNET =
            "There is no student of with this NUSNet ID!";
```

Reply:
Implemented in e6bf6760c4ee6ccd0246bec9b03915d147c6c2b2